spec/requests/api/v1/notifications/requests_spec.rb

```diff
@@ -120,4 +120,38 @@
       expect(response).to have_http_status(200)
     end
   end
+
+  describe 'GET /api/v1/notifications/requests/merged' do
+    subject do
+      get '/api/v1/notifications/requests/merged', headers: headers
+    end
+
+    it_behaves_like 'forbidden for wrong scope', 'write write:notifications'
+
+    context 'when the user has no accepted request pending merge' do
+      it 'returns http success and returns merged: true' do
+        subject
+
+        expect(response).to have_http_status(200)
+        expect(body_as_json).to include(
```

Comment:
Why not match this exactly?

The code cleary returns an object with a single key. The spec(ification) should maybe reflect that.

Reply:
I went for consistency with other tests which use `include` everywhere, but this makes sense. Changed to test equality.